examples/datasets/plot_brainstorm_data.py

```diff
@@ -23,18 +23,18 @@
 import numpy as np
 
 import mne
-from mne.datasets.brainstorm import bst_raw
+from mne.datasets.brainstorm import bst_auditory
+from mne.io import read_raw_ctf
 
 print(__doc__)
 
 tmin, tmax, event_id = -0.1, 0.3, 2  # take right-hand somato
 reject = dict(mag=4e-12, eog=250e-6)
 
-data_path = bst_raw.data_path()
+data_path = bst_auditory.data_path()
```

Comment:
Try: 

http://neuroimage.usc.edu/bst/getupdate.php?u=mne&s=sample_auditory
http://neuroimage.usc.edu/bst/getupdate.php?u=mne&s=sample_raw

etc.

Reply:
the second is the file. Thx.